src/bitmessagekivy/tests/test_file_manager.py

```diff
@@ -0,0 +1,117 @@
+            '//Random//RandomBoxlayout//MDFillRoundFlatIconButton[@text=\"Proceed Next\"]', timeout=3)
+        # Checking "My Address" Screen after creating a address
+        self.assertExists("//ScreenManager[@current=\"myaddress\"]", timeout=5)
+        # Checking the new address is created
+        self.assertExists('//MyAddress//MDList[0]/CustomTwoLineAvatarIconListItem', timeout=10)
+
+    @ordered
+    def test_set_default_address(self):
+        """Select First Address From Drawer-Box"""
+        # Checking current screen
+        self.assertExists("//ScreenManager[@current=\"myaddress\"]", timeout=5)
+        # This is for opening side navbar
+        self.open_side_navbar()
+        # Click to open Address Dropdown
+        self.assertExists('//NavigationItem[0][@text=\"dropdown_nav_item\"]', timeout=2)
+        self.assertExists(
+            '//NavigationItem[0][@text=\"dropdown_nav_item\"]'
+            '/IdentitySpinner[@name=\"identity_dropdown\"]', timeout=1
+        )
+        is_open = self.cli.getattr(
+            '//NavigationItem[0][@text=\"dropdown_nav_item\"]'
+            '/IdentitySpinner[@name=\"identity_dropdown\"][@is_open]', "is_open")
+        self.assertEqual(is_open, False)
+        self.cli.wait(
+            '//NavigationItem[0][@text=\"dropdown_nav_item\"]'
+            '/IdentitySpinner[@name=\"identity_dropdown\"][@state=\"normal\"]', timeout=2
+        )
+        self.cli.wait_click(
+            '//NavigationItem[0][@text=\"dropdown_nav_item\"]'
+            '/IdentitySpinner[@name=\"identity_dropdown\"]', timeout=1
+        )
+        if self.cli.wait(
+            '//NavigationItem[0][@text=\"dropdown_nav_item\"]'
+                '/IdentitySpinner[@name=\"identity_dropdown\"][@state=\"normal\"]', timeout=2):
+            sleep(0.2)
+        # Check the state of dropdown.
+        is_open = self.cli.getattr(
+            '//NavigationItem[0][@text=\"dropdown_nav_item\"]'
+            '/IdentitySpinner[@name=\"identity_dropdown\"][@is_open]', 'is_open'
+        )
+        self.assertEqual(is_open, True)
+        # List of addresses
+        addresses_in_dropdown = self.cli.getattr(
+            '//NavigationItem[0][@text=\"dropdown_nav_item\"]/IdentitySpinner[@values]', 'values'
+        )
+        # Checking the dropdown options are exists
+        self.assertGreaterEqual(len(self.cli.getattr(
+            '//MySpinnerOption[@text]', 'text')), len(addresses_in_dropdown)
+        )
+        # Selection of an address to set as a default address.
+        self.cli.wait_click('//MySpinnerOption[0]', timeout=5)
+
+    @ordered
+    def test_open_file_manager(self):
+        """Opening and Closing File manager."""
+        # Clicking on file manager icon
+        self.cli.wait_click('//ContentNavigationDrawer//MDIconButton[1]', timeout=1)
+        # Checking the state of file manager is it open or not
+        if self.cli.execute('app.file_manager_open'):
+            self.cli.sleep(3)
```

Comment:
This should be done differently. At the very least, there are no asserts so it doesn't actually test anything. Sleep should be avoided too, or at least it should be shorter. The video recording is 60 FPS so there is no need to wait for 3 seconds.